src/Individual Article Page/ArticlePage.jsx

```diff
@@ -69,14 +73,27 @@ const ArticlePage = () => {
     setCommentStatus(false);
   };
 
+  const handleDeleteClick = (comment_id) => {
+    setDeleteCommentId(comment_id)
+    setDeleted(true)
```

Comment:
As we have this state keeping track of when something is deleted, it might be nice to return some HTML which confirms the comment has been deleted 👍